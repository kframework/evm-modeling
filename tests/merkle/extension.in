MerkleUpdate( MerkleUpdate( .MerkleTree, "dog", "puppy" ), "doge", "coin" )
