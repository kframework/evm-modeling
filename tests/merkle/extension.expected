<k>
  MerkleExtension ( b"\x00dog" , MerkleBranch ( 0 |-> .MerkleTree
  1 |-> .MerkleTree
  2 |-> .MerkleTree
  3 |-> .MerkleTree
  4 |-> .MerkleTree
  5 |-> .MerkleTree
  6 |-> MerkleLeaf ( b"5" , "coin" )
  7 |-> .MerkleTree
  8 |-> .MerkleTree
  9 |-> .MerkleTree
  10 |-> .MerkleTree
  11 |-> .MerkleTree
  12 |-> .MerkleTree
  13 |-> .MerkleTree
  14 |-> .MerkleTree
  15 |-> .MerkleTree , "puppy" ) )
</k>
